spec/views/content/shared/funding.html.erb_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+describe "_funding.html.erb" do
+  let(:subject_name) { "chemistry" }
+  let(:salary_value) { YAML.load_file("config/values/salaries.yml").dig("burseries", "postgraduate", "chemistry") }
+
+  subject do
+    assign(:front_matter, { "subject" => subject_name })
+    assign(:bursaries_postgraduate_chemistry, "£25,000")
```

Comment:
This is obviously not the right amount for the current chemistry bursary - and scholarship in the line below - does that matter? I'm presuming this isn't something we'd need to maintain as bursaries and scholarships changed @sarahcrack ?